engine/src/flutter/display_list/testing/dl_test_snippets.cc

```diff
@@ -753,17 +753,9 @@ std::vector<DisplayListInvocationGroup> CreateAllRenderingOps() {
            {1, 56, 1,
             [](DlOpReceiver& r) { r.drawRoundRect(kTestRRect.Shift(5, 5)); }},
        }},
-      {"DrawRSuperellipse",
-       {
-           {1, 56, 1,
-            [](DlOpReceiver& r) {
-              r.drawRoundSuperellipse(kTestRSuperellipse);
-            }},
-           {1, 56, 1,
-            [](DlOpReceiver& r) {
-              r.drawRoundSuperellipse(kTestRSuperellipse.Shift(5, 5));
-            }},
-       }},
+      // DrawRSuperellipse is omitted because the testing framework doesn't
```

Comment:
What about the ClipRSE entries? Oh, I guess those are always filled so they won't ever have "2 versions".

Reply:
Yeah. They always uses the geometry. :)